acceptance/server_test.go

```diff
@@ -101,19 +104,23 @@ func AddHandlers(server *testserver.Server) {
 	server.Handle("POST", "/api/2.0/workspace/delete", func(req testserver.Request) any {
 		path := req.URL.Query().Get("path")
 		recursive := req.URL.Query().Get("recursive") == "true"
+		log.Debugf(context.Background(), "[workspace files] Delete file: %s, %v ", path, recursive)
```

Comment:
Why do we need to log this and below? We already have a record and if you need to debug you can put a breakpoint + inprocess mode.

Reply:
A minimal test makes around 20 operations with files in a workspace. For me it was useful to review an ordered list of changes to debug an issue rather than breaking the program execution 20 times while keeping in mind what the previous operations were.

I do not insist on this change, if you have a strong opinion this should not be here, I can remove the log lines, but it was useful for me to debug a test behavior, thus I think it could be useful for other contributors in the future.

Reply:
> A rather small test makes around 20 operations with files in a workspace.

Did you try adding --debug to $CLI ?

>  I can remove the log lines, but it was useful for me to debug a test behavior

I'm curious if --debug helps. If it does not, we can add the logging, but it should use t.Log and it should be a separate PR. Maybe it should also be behind a separate flag.